Ch10/Ch10.md

```diff
@@ -528,21 +530,27 @@ Now, let's do some queries on this index.
 #### Full text search queries
 
 Querying a full text search field is an interesting experience because what we think we're doing and what's actually happening is so often drastically 
-different, yet the end result is the same. Consider the query `from index 'test/search' as n where search(n.Search, "princess flower tower")`.
+different, yet the end result is the same. 
+Consider the query:  
+`from index 'test/search' as n  
+where search(n.Search, "princess flower tower")`.
 
 If you run this query, you'll find that this actually matches the document, even though the word `flower` is nowhere to be found. That's because
 of the way RavenDB processes queries on full text search. It's enough that we have _any_ match to be considered enough to return the result. More 
 advanced options, such as phrase queries, are also available when using Lucene directly, such as the following queries:
 
-* `where lucene(n.Search, ' "princess tower" ')`&mdash;phrase query match (note the `"` in the query) because there's a `princess` followed by a `tower` in 
+* `where lucene(n.Search, "princess tower")` &mdash;  
```

Comment:
No, here, the ' "term " ` is intentional and really required

Reply:
done